### PR TITLE
QMAPS-759 display POI image on top right if available

### PR DESCRIPTION
**Pull request by xem**

### CODE REVIEW

File: src/panel/poi_bloc/images_panel.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import ImagesPanelView from '../../views/poi_bloc/images.dot'
+import nextTransitionPartial from '../../views/poi_bloc/hour_next_transition_partial.dot'
+import Panel from "../../libs/panel";
+import OsmSchedule from '../../../src/adapters/osm_schedule'
+import Telemetry from "../../libs/telemetry";
+
+function ImagesPanel(block, poi) {
```
Comment (Julien-laville) [inline src/panel/poi_bloc/images_panel.js]: poi is not used

File: src/panel/poi_bloc/images_panel.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+  console.log(block);
```
Comment (Julien-laville) [inline src/panel/poi_bloc/images_panel.js]: to remove

File: src/panel/poi_bloc/images_panel.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  this.nextTransitionPartial = nextTransitionPartial
```
Comment (Julien-laville) [inline src/panel/poi_bloc/images_panel.js]: not used

File: src/panel/poi_bloc/images_panel.js [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+  this.panel = new Panel(this, ImagesPanelView)
+  this.name = block.name
+  this.block = block
+  this.latLng = poi.latLon
```
Comment (Julien-laville) [inline src/panel/poi_bloc/images_panel.js]: not used

File: src/panel/poi_bloc/images_panel.js [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+  this.isCollapsed = true
```
Comment (Julien-laville) [inline src/panel/poi_bloc/images_panel.js]: not used

File: src/panel/poi_bloc/images_panel.js (hunk 1 above)
Comment (Julien-laville) [inline src/panel/poi_bloc/images_panel.js]: please use the new syntax
export class ImagePanel
then add a constructor

File: src/panel/poi_bloc/images_panel.js [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+}
+
+export default ImagesPanel
```
Comment (Julien-laville) [inline src/panel/poi_bloc/images_panel.js]: to remove

File: src/views/poi_bloc/images.dot [hunk 7]
```diff
@@ -0,0 +1 @@
+<div class="poi_panel__image" style="background-image: url('{{= this.block.images[0].url }}')"></div>
```
Comment (Julien-laville) [inline src/views/poi_bloc/images.dot]: you should extract image data in the panel code

File: src/panel/poi_bloc/images_panel.js (hunk 1 above)
Comment (xem) [reply src/panel/poi_bloc/images_panel.js]: which new syntax?  ES6?

**Comment by jbgriesner** [comment]
Thanks for these reviews @Julien-laville !

**Comment by GuillaumeGomez** [comment]
I have a JS failure when clicking on this element:

Screenshot from 2019-04-11 10-50-21

The JS error is the following:

```
{…}
​
exception: 0
​
file: "idunn_poi"
​
message: "unknown error getting idunn poi reaching https://maps.dev.qwant.ninja/maps/detail/v1/places/osm:node:4862223479 with options {\"lang\":\"en\"}"
​
method: "poiApiLoad"
​
<prototype>: Object { … }
error.js:41:4
    sendOnce error.js:41
    _callee$ idunn_poi.js:55
    Babel 8
```

**Comment by GuillaumeGomez** [comment]
Is there any POI with an image? Even the Eiffel tower doesn't have one...

File: config/constants.yml [hunk 8]
```diff
@@ -101,6 +99,8 @@ pois:
   panelName: internet_access
 - apiName: contact
   panelName: contact
+- apiName: images
+  panelName: images
```
[the remark this answers is not in the document]
Comment (GuillaumeGomez) [reply config/constants.yml]: It won't be unused for long so I'd be in favor to keep it for now.

File: src/views/app_panel.dot [hunk 9]
```diff
@@ -1,5 +1,5 @@
 <div class="side_panel__container">
-  <div class="favorite_poi_panel__container">
+  <div class="favorite_poi_panel__container" oncontextmenu="function(e){e.preventDefault();return false;}">
```
[the remark this answers is not in the document]
Comment (GuillaumeGomez) [reply src/views/app_panel.dot]: To prevent events on the panel to be sent to the map behind.

**Pull request merged**